packages/components/src/components/radio-group/radio-group.tsx

```diff
@@ -2,7 +2,13 @@ import React, { ChangeEvent } from 'react';
 import classNames from 'classnames';
 import Text from '../text';
 
-type TItem = React.HTMLAttributes<HTMLDivElement> & { id?: string; value: string; label: string; disabled: boolean };
+type TItem = React.HTMLAttributes<HTMLDivElement> & {
+    id?: string;
+    value: string;
+    label: string;
+    disabled: boolean;
+    is_hidden?: boolean;
```

Comment:
Don't you need to give this a falsy default value so that the rest of the usage won't fail? (better to false than undefined in line no.55)

As you work on updating the shared component, please remember to incorporate the QA checklist for all the areas where this component has been utilised.